src/plugins/data_source/common/credentials/types.ts

```diff
@@ -10,6 +10,7 @@ import { SavedObjectAttributes } from 'src/core/types';
  */
 export enum CredentialMaterialsType {
   UsernamePasswordType = 'username_password',
+  NoAuth = 'no_auth',
```

Comment:
we have moved no_auth to data source, not credential

Reply:
so there will be only one option ? Username  & Password?

Reply:
Yes for current release.